ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-main__ocp-cnv-odf-4.11.yaml

```diff
@@ -33,7 +33,7 @@ resources:
       memory: 200Mi
 tests:
 - as: cnv-odf-tests-aws-upi-ocp411
-  interval: 960h
+  interval: 168h
```

Comment:
how about using cron instead of interval?

[cron: 0 0 * * 1](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml#LL113C3-L113C18)
or
[cron: '@weekly'](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml#LL196C3-L196C18)